tests/projects/test_projects_cli.py

```diff
@@ -103,3 +105,24 @@ def test_run_databricks_cluster_spec(tmpdir):
                       json.dumps(cluster_spec) + "JUNK", "-e", "greeter", "-P", "name=hi"],
             env={'MLFLOW_TRACKING_URI': 'databricks://profile'})
         assert res.exit_code != 0
+
+
+@pytest.mark.parametrize('experiment_name,experiment_id,expected', [
```

Comment:
er, i didn't realize these lived in test_projects.py! sorry do you mind moving them back (with your added test cases of course). please feel free to call me out if i don't make full sense :-) thanks!

Reply:
Cool should be good to go!